Makefile

```diff
@@ -141,7 +144,7 @@ generate-swagger:
 swagger-check: generate-swagger
 	@diff=$$(git diff '$(SWAGGER_SPEC)'); \
 	if [ -n "$$diff" ]; then \
-		echo "Please run 'make generate-swagger' and commit the result:"; \
+		echo "Please run '$(MAKE) generate-swagger' and commit the result:"; \
```

Comment:
Why use $(MAKE) for echo statements here?  Makes sense when running code but seems to unnecessarily add full path to the echo statements which seems more confusing:
```
Please run '/Applications/Xcode.app/Contents/Developer/usr/bin/make generate-swagger' and commit the result:'
```
We don't change the default $(MAKE) either so it should always be the same as make. Maybe there is some other benefit to this?

Reply:
The big issue is really the first (line 10ish), which causes the build to produce the wrong version number on FreeBSD. The others were informational and a minor annoyance at worst. I'm good with leaving only the first one in, and reverting the others if that's what people want.

Reply:
Yea that first one makes sense. I'd prefer to revert the other changes in the echo statements for the reason stated above

Reply:
Sounds good, I'll put a new commit up shortly.